modules/mobile/spec/request/debts_request_spec.rb

```diff
@@ -53,4 +55,62 @@
       end
     end
   end
+
+  describe 'GET /mobile/v0/debts/:id' do
+    let(:debt_id) { '2b41479f-2181-4016-9f77-3359c3ecee74' }
+
+    let(:load_debt_store) do
+      debts = [{ 'fileNumber' => '796043735',
+                 'payeeNumber' => '00',
+                 'personEntitled' => nil,
+                 'deductionCode' => '30',
+                 'benefitType' => 'Comp & Pen',
+                 'diaryCode' => '914',
+                 'diaryCodeDescription' => 'Paid In Full',
+                 'amountOverpaid' => 123.34,
+                 'amountWithheld' => 50.0,
+                 'originalAR' => 1177.0,
+                 'currentAR' => 123.34,
+                 'debtHistory' =>
+                  [{ 'date' => '09/12/1998', 'letterCode' => '123',
+                     'description' => 'Third Demand Letter - Potential Treasury Referral' }] }]
+
+      debts.map { |d| d['id'] = SecureRandom.uuid }
+      debts[0]['id'] = debt_id
```

Comment:
I'm not sure I understand the two lines above. You seem to be giving each debt record (there's only one) an id, then you manually give the one record an id again. Was that map from an earlier version?

Reply:
yeah, this was copied from the VA.gov controller and I simplified it. Corrected